<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>472</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>472</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mood Driven Music Players</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>PLAYER CONTROL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QListWidget" name="listWidged_playlist">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>301</width>
       <height>261</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
    <widget class="QLabel" name="image_cover">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>40</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Resources.qrc">:/images/Resources/cover_missing.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_play">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>155</y>
       <width>41</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_next">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_prev">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mediastate">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>190</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>No Music</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_moodFlip">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>220</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: green;
</string>
     </property>
     <property name="text">
      <string>MOOD
FLIP</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>215</y>
       <width>120</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>image_cover</zorder>
    <zorder>label_mediastate</zorder>
    <zorder>line_6</zorder>
    <zorder>frame</zorder>
    <zorder>listWidged_playlist</zorder>
    <zorder>btn_moodFlip</zorder>
    <zorder>button_play</zorder>
    <zorder>button_prev</zorder>
    <zorder>button_next</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>ENGINE STATUS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>System Status:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_engineStatus">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Not connected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_disconnect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>MOOD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="image_mood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Resources.qrc">:/images/Resources/unknown.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuPlaylist_source">
     <property name="title">
      <string>Music Source</string>
     </property>
     <addaction name="actionLocal"/>
     <addaction name="actionStereomood"/>
    </widget>
    <addaction name="actionSet_Playlist"/>
    <addaction name="menuPlaylist_source"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionTo_EmoEngine"/>
    <addaction name="actionTo_EmoComposer"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStatus_Console"/>
   </widget>
   <addaction name="menuMen"/>
   <addaction name="menuConnect"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSet_Playlist">
   <property name="text">
    <string>Set Playlist</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLocal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local</string>
   </property>
  </action>
  <action name="actionStereomood">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stereomood</string>
   </property>
  </action>
  <action name="actionTo_EmoEngine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>To EmoEngine</string>
   </property>
  </action>
  <action name="actionTo_EmoComposer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To EmoComposer</string>
   </property>
  </action>
  <action name="actionStatus_Console">
   <property name="text">
    <string>Status Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
